src/RequestBodyParser.php

```diff
@@ -114,4 +115,19 @@
         }
         return null;
```

Comment:
I think that such a check does not make sense, and it can be deleted

Reply:
Let's keep it for now. It doesn't change API or anything if deleted.